cves/CVE-2016-7905.yml

```diff
@@ -196,8 +214,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    Using static analysis tools would be able to find this Null Pointer
```

Comment:
To make this more broad could you provide a small description of a static analysis tool?